src/wpool_queue_manager.erl

```diff
@@ -348,6 +402,15 @@ handle_call(worker_counts, _From, State) ->
 
 %% @private
 -spec handle_info(any(), state()) -> {noreply, state()}.
+handle_info({'DOWN', _Ref, process, Worker, Exit}, State = #state{monitors = Mons}) ->
```

Comment:
Line 405 is too long: handle_info({'DOWN', _Ref, process, Worker, Exit}, State = #state{monitors = Mons}) ->.